docs/azure/extensions.md

```diff
@@ -9,10 +9,24 @@ DateApproved: 02/1/2024
+
+|Type of Developer|Description|When to use|
+|----------------------|---------------------|-------------|
+|Fullstack developer|Developers who work on both frontend and backend aspects of apps.|**Azure Container Apps:** Create or manage containerized apps that handle various parts of their stack.</br> **Azure Functions:** Create serverless functions to handle backend tasks and integrate with frontend apps.</br> **App Service:** Deploy complete web solutions, including dynamic backend services, APIs, and server-side logic.|
+|Backend developer|Developers building microservices architectures and leveraging containers to encapsulate/manage individual microservices.|**Azure Container Apps:** Server-side logic, APIs, and microservices</br> **Azure Functions:** Server-side logic and APIs</br> **Azure Container Apps:** Deploying and managing containerized backend services</br> **Azure Functions:** Build scalable, event-driven backend services without managing infrastructure</br> **App Service:** Deploy RESTful APIs, microservices or other backend components|
+|DevOps Engineer|Engineers responsible for setting up or maintaining CI/CD pipelines and managing cloud infrastructure.|**Azure Container Apps:** Automate the deployment/scaling of containerized applications</br> **Azure Functions:** Automate the deployment of serverless functions and integrate them into a broader DevOps workflow</br> **App Service:** Integrate code repositories with automated deployment processes and monitor application performance and logs</br> **Static Web Apps:** Automate the deployment process and ensure seamless integration with GitHub Actions or Azure DevOps|
+|Frontend developer|Developers who specialize in building user interfaces using frameworks and libraries like React, Angular, Vue.js, or plain HTML/CSS/JavaScript.|**Static Web Apps**, **App Service**, **Storage**, **Azure Functions:** A streamline workflow to develop, preview, and deploy static websites and single-page apps|
+|Enterprise Developer|Developers working on large-scale apps, services for businesses, and building Software as a Service (SaaS) apps.|**Azure Container Apps**, **Azure Functions**, **App Service:** Deployment, scaling, and monitoring capabilities for mission-critical apps</br> Using containers to create scalable, multi-tenant services and automate various backend processes|
+|API developer|Creates and manages APIs for various clients and apps|**Azure Functions** or **Azure Container Apps:** Develop and deploy serverless APIs</br> **Storage:** API data storage needs (blobs, queues, tables, and files, upload and download data)|
+|Data engineer|Processes and analyzes data streams|**Azure Container Apps:** Deploy data processing and machine learning models in containers, Scalable and reproducible environments for their data-driven apps</br> **Azure Functions:** Trigger data processing workflows, handle ETL tasks, and respond to data changes in real-time|
+
 ## Visual Studio Code Marketplace
```

Comment:
@ntrogh I need to edit this section (add, remove links) but I need some guidance here how to do that.

Reply:
@mistymadonna this section is part of the website (private) repo. I can help with that.